Contents/mods/FuelAPI/media/lua/shared/Translate/PL/IG_UI_PL.txt

```diff
@@ -0,0 +1,3 @@
+IG_UI_PL = {
+    IGUI_BarrelHasFuel = "Posiada benzyn� w �rodku.",
```

Comment:
IGUI_BarrelHasFuel = "Zawiera benzynę.",

Reply:
special letters are wrong coded in your wersion @Tread4